frontend/packages/git-service/src/services/gitlab-service.ts

```diff
@@ -48,33 +69,21 @@ export class GitlabService extends BaseService {
 
   getProjectId = async (): Promise<any> => {
     try {
-      const resp = await this.client.Projects.search(this.metadata.repoName);
-      for (const re of resp) {
-        if (re.path_with_namespace === this.metadata.fullName) {
-          return re.id;
-        }
-      }
-      throw new Error('Unable to find gitlab project id, check project details!');
+      const repo = await this.getRepo();
+      return repo.id;
     } catch (e) {
       throw e;
     }
   };
 
   isRepoReachable = async (): Promise<boolean> => {
     try {
-      const resp = await this.client.Projects.search(this.metadata.repoName);
-      if (resp.length < 1) {
-        return false;
-      }
-      for (const re of resp) {
-        if (re.path_with_namespace === this.metadata.fullName) {
-          return true;
-        }
-      }
-
-      return false;
+      await this.getRepo();
+      return true;
     } catch (e) {
-      throw e;
+      // eslint-disable-next-line no-console
+      console.error(e);
```

Comment:
Do we really need to log this error in console?

Reply:
@christianvogt does it make sense to release the product with console logs? I am not sure. Probably we should look into having a proper catch and throw routine for such cases.

Reply:
No, we probably don't need the console log... but catching an exception just to throw it does nothing. I figured if it happens we could put a log into the console to help understand that the `false` to `isRepoReachable` was because of a critical issue. It's not visible to the end-user either way. Uncaught throws cause a log error anyways, this way it's just handled more gracefully.

Reply:
I find most logs to just be noise because we don't have a standard way of dealing with it.
Try catch is a standard way of handling rejected promises with async await. I don't think we need to console log in this case.